web-portal/components/admin-venue-edit/VenueCard.vue

```diff
@@ -224,11 +258,24 @@
         const fieldName = event.target.name
         const newValue = event.target.value
 
-        const venue = {
+        let venue = {
           ...this.venue,
           [fieldName]: newValue
         }
 
+        if (['street', 'city', 'state', 'zip', 'neighborhood'].includes(fieldName)) {
+          venue = {
+            ...venue,
+            'address': [
+              venue.street,
+              venue.city,
+              venue.state,
+              venue.zip,
+              venue.neighborhood
+            ].filter(a => a).join(', ')
+          }
+        }
+
```

Comment:
I think this is a fine way of handling this, especially since we expect it to be a temporary arrangement. If you're concerned about readability, two things to consider would be

- a comment explaining that `address` is the deprecated composite of the other address fields
- extract the `filter`-`join` construct to another method or utility function with a name like `constructAddressFromFields`